tests/wdio/utilities-spec.js

```diff
@@ -26,8 +26,6 @@ Terra.describeViewports('utilities', ['huge'], () => {
     browser.click('#terra-dev-site-locale-select');
     browser.click('#terra-select-option-en-AU');
 
-    browser.click('#terra-dev-site-theme-select');
```

Comment:
Removed, now that we're relying on the `TERRA_THEME_CONFIG` global, it changes the available themes when tests are run with an override theme. So this select menu is not reliably available for all tests.